examples/emcute_mqttsn/README.md

```diff
@@ -105,3 +105,18 @@ default, this example sets this statically ID to `gertrud`. If you want to
 connect more than one node to the broker, you need to set a custom ID for each
 node during compile time. Simply use the `EMCUTE_ID` environment variable for
 this, e.g. build with `EMCUTE_ID=horst make all`.
+
+### I see incomming messages in RSMB, but no outgoing message seems to arrive?
+The UDP socket handling for IPv6 based endpoints in the `Mosquitto.rsmb`
+implementation is buggy when it comes to handling link local addresses,
+as the implementation does not remember the interface on which data comes in,
+hindering it from sending out any responses.
+
+Quick workaround: simply use global addresses
```

Comment:
Minor nit: I would formulate a bit more concise, i.e.

Workaround: either use global IPv6 addresses or [ULAs](https://tools.ietf.org/html/rfc4193).

Reply:
Seems like a good idea. I think most important is the explanation why messages get lost. My own workaround was to alter the routing on the linux machine running RSMB.
Is there documentation for a beginner friendly way to add a new address? Maybe it should be linked here too. I found only this makefile flag which is mentioned in this issue https://github.com/RIOT-OS/RIOT/issues/11468 .